odl/phantom/transmission.py

```diff
@@ -75,6 +75,11 @@ def _modified_shepp_logan_ellipses(ellipses):
     """
     intensities = [1.0, -0.8, -0.2, -0.2, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
 
+    # Add minimal numbers to ensure that the result is nowhere negative.
+    # This is needed due to numerical issues.
+    intensities[2] += 5e-17
+    intensities[3] += 5e-17
```

Comment:
I really hate that it's impossible on GitHub to simply look a bit farther up in these snippets. So goddamn hard to know in which  function you are, and because I'm lazy, I just guess.
In the emails you get it's actually much better.

Anyway, if the fix does the trick then whatever.